Source/MultipartFormData.swift

```diff
@@ -256,16 +256,17 @@ public class MultipartFormData {
 
         var isReachable = false
 
-        do {
-            try fileURL.checkResourceIsReachable()
+        if #available(OSX 10.10, *) {
```

Comment:
@cnoon Is Alamofire 2.0 when official Swift 2 support will be landed? If official support for Swift 2 will land before then, it seems like supporting iOS 7 until the 2.0 release is necessary. So far, the changes to support iOS 7 in the `swift-2.0` branch have only been a single (other) `#available` check. While I'm not usually one to want (or need) much legacy support, I am in the middle of a project that has to target iOS 7 while using Swift 2 and it would be unfortunate to be stuck on an old version of Alamofire.